qiskit_ionq/helpers.py

```diff
@@ -244,7 +236,7 @@ def qiskit_circ_to_ionq_circ(
             converted["gate"] = instruction_name
 
         # Make sure uncontrolled multi-targets use all qargs.
-        if isinstance(instruction, multi_target_uncontrolled_gates):
+        if instruction.num_qubits > 1 and not hasattr(instruction, "num_ctrl_qubits"):
```

Comment:
is num_qubits guaranteed to be set?

I don't quite understand why we are changing this and not https://github.com/qiskit-community/qiskit-ionq/blob/08a15dc4ae19880533d0732d9eab14766fe9341a/qiskit_ionq/helpers.py#L193-L194 based on the gates you've seen issues with?

Reply:
Spoke offline: main issue is that where I linked we check gate name, and here we are checking python classes, which might not match in the case of QASM/etc

WDYT about changing to the values from GATESET_MAP that we know should have multiple controls?

This LGTM assuming the num_qubits attribute must exist, if it might not I would change to getattr(instruction, "num_qubits", 0)